### PR TITLE
feat: Swift API for JSON Support setJSON

### PR DESCRIPTION
**Pull request by jayahariv**
* add initializer and setJSON functions for the swift classes.

### CODE REVIEW

File: Swift/MutableDocument.swift [hunk 1]
```diff
@@ -70,6 +80,18 @@ public final class MutableDocument : Document, MutableDictionaryProtocol {
         setData(data)
     }
     
+    /// Initializes a new CBLMutableDocument object with the given ID and the JSON data.  If a
```
Comment (pasin) [inline Swift/MutableDocument.swift:83]: CBLMutableDocument -> MutableDocument

File: Swift/MutableDocument.swift [hunk 2]
```diff
@@ -54,6 +54,16 @@ public final class MutableDocument : Document, MutableDictionaryProtocol {
         setData(data)
     }
     
+    /// Initializes a new CBLMutableDocument object with the JSON data.
```
Comment (pasin) [inline Swift/MutableDocument.swift:57]: CBLMutableDocument -> MutableDocument

**Review by pasin — APPROVED** [review]

File: Swift/MutableDocument.swift (hunk 2 above)
Comment (jayahariv) [inline Swift/MutableDocument.swift:57]:
```suggestion
    /// Initializes a new MutableDocument object with the JSON data.
```

File: Swift/MutableDocument.swift (hunk 1 above)
Comment (jayahariv) [inline Swift/MutableDocument.swift:83]:
```suggestion
    /// Initializes a new MutableDocument object with the given ID and the JSON data.  If a
```

**Pull request merged**